src/main/java/net/minecraftforge/common/extensions/IForgeSpreadingBlock.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * Minecraft Forge
+ * Copyright (c) 2016-2021.
+ *
+ * This library is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation version 2.1
+ * of the License.
+ *
+ * This library is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this library; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
+ */
+
+package net.minecraftforge.common.extensions;
+
+import net.minecraft.core.BlockPos;
+import net.minecraft.server.level.ServerLevel;
+import net.minecraft.world.level.block.state.BlockState;
+import net.minecraftforge.common.block.spreader.SpreadBehaviors;
+import net.minecraftforge.common.block.spreader.SpreaderType;
+
+import java.util.Random;
+
+public interface IForgeSpreadingBlock
+{
+
+
+    default void spread(BlockState state, ServerLevel level, BlockPos pos, Random random, int tries, int range){
```

Comment:
Space after the closing parenthesis of a method definition, and opening brace should be on a newline:
```suggestion
    default void spread(BlockState state, ServerLevel level, BlockPos pos, Random random, int tries, int range)
    {
```